js/pages/samples.js

```diff
@@ -405,6 +405,12 @@ const henry_ford = {
           'Monday to Friday 8:00 a.m. - 4:15 p.m. <br> Saturday 7:00 a.m. - 12:00 p.m.',
           'Parking lots are conveniently located near all hospital entrances for those who do not wish to use valet. Wheelchairs are available at all entrances and handicapped parking is available in each parking area. Valet parking is available at all hospital entrances for $3.00. This service is not available on holidays.',
         ],
+        [
+           'Henry Ford Hospital-West Bloomfield',
+           '6777 West Maple Road<br>West Bloomfield, MI 48322<br>Location: Lab Services First Floor-Bloomfield Reception',
```

Comment:
Can you confirm this is grammatically correct?
Current: `Location: Lab Services First Floor-Bloomfield Reception.` (hyphen vs dash)
Possibly: `Location: Lab Services First Floor - Bloomfield Reception.`